packages/adapter-flagsmith/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import type { Adapter } from 'flags';
+import flagsmith, { IFlagsmithFeature, IInitConfig } from 'flagsmith';
+
+type AdapterParams = {
+  key?: string;
+} & IInitConfig;
+
+export function createFlagsmithAdapter<
+  ValueType extends IFlagsmithFeature,
+  EntitiesType,
+>({
+  key: customKey,
+  ...configParams
+}: AdapterParams): Adapter<ValueType, EntitiesType> {
+  async function initialize(config: IInitConfig) {
+    if (flagsmith?.initialised) return;
+    await flagsmith.init({ fetch: global.fetch, ...configParams });
+  }
+
+  return {
+    async decide({
```

Comment:
Adapters can provide `origin` here, which is a hyperlink to the feature flag defined in code. If you can provide this hyperlink, it will allow users to get to the right place in one click from the Flags Explorer

https://flags-sdk.dev/frameworks/next#flags-explorer

Ex. something like

```typescript
return { 
  origin: `${configParams.api}/${configParams.environmentId}/feature/${key}`
  decide,
}
```